### PR TITLE
Expand documentation and notebooks.

### PR DESCRIPTION
**Pull request by KonstantinWilleke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ Below we provide a step-by-step guide for getting started with the competition.
 
 ## 2. Download neural data
 
-You can download the data from [https://gin.g-node.org/cajal/Sensorium2022](https://gin.g-node.org/cajal/Sensorium2022) and unzip it into `sensorium/notebooks/data`
+You can download the data from [https://gin.g-node.org/cajal/Sensorium2022](https://gin.g-node.org/cajal/Sensorium2022) and place it in `sensorium/notebooks/data`.
+**Note:** Downloading the files all at once as a directory does lead to unfortunate errors. Thus, all datastes have to be downloadede individually.
```
Comment (mohammadbashiri) [inline README.md:32]: "downloadede" -> "downloaded"

**Review by mohammadbashiri — CHANGES_REQUESTED** [review]

File: notebooks/model_tutorial/1a_model_training_sensorium.ipynb [hunk 2]
```diff
@@ -8,7 +8,7 @@
     }
```
Comment (mohammadbashiri) [inline notebooks/model_tutorial/1a_model_training_sensorium.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Our LN model has the same architecture as our CNN model (a convolutional core followed by a gaussian readout)
</pre><p>Is there a reason this is a comment in the cell and not a markdown cell before this cell? If there is no specific reason, could you put the text into a mardown cell before this cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

File: notebooks/model_tutorial/0_baseline_CNN.ipynb [hunk 3]
```diff
@@ -2,44 +2,68 @@
  "cells": [
```
Comment (mohammadbashiri) [inline notebooks/model_tutorial/0_baseline_CNN.ipynb:2]: <p>".. look at the .."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

File: notebooks/model_tutorial/1b_model_training_sensorium+.ipynb [hunk 4]
```diff
@@ -8,7 +8,7 @@
     }
```
Comment (mohammadbashiri) [inline notebooks/model_tutorial/1b_model_training_sensorium+.ipynb:8]: <p>Same here with the comment vs markdown file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: notebooks/submission_tutorial/1_sensorium_submission.ipynb [hunk 5]
```diff
@@ -0,0 +1,495 @@
+{
```
Comment (mohammadbashiri) [inline notebooks/submission_tutorial/1_sensorium_submission.ipynb:1]: <p>".. normalized by its own STD" -&gt; ".. divided by its own STD (which is computed across all images in the train set)".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (mohammadbashiri) [inline notebooks/submission_tutorial/1_sensorium_submission.ipynb:1]: <p>repeated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (mohammadbashiri) [inline notebooks/submission_tutorial/1_sensorium_submission.ipynb:1]: <p>repeated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (mohammadbashiri) [inline notebooks/submission_tutorial/1_sensorium_submission.ipynb:1]: <p>repeated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

File: notebooks/submission_tutorial/2_behind_the_scenes.ipynb [hunk 6]
```diff
@@ -0,0 +1,527 @@
+{
```
Comment (mohammadbashiri) [inline notebooks/submission_tutorial/2_behind_the_scenes.ipynb:1]: <p>Let's changed "show" -&gt; "demo". Or somehow make it clear that "to demo this we will use the pre-training dataset for which the test responses are available".</p><p>It's probably due to my stupidity, but when I read this first I was wondering what does the pre-training dataset have to do with the competition, and then I realized it's just an example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

File: notebooks/dataset_tutorial/README.md [hunk 7]
```diff
@@ -1,21 +1,62 @@
...
+  - **responses:** This directory contains NumPy arrays where each single `X.npy` contains the deconvolved calcium traces (i.e. responses) recorded from the mouse in trial `X` in response to the particular presented image.
+  - **behavior:** Behavioral variables include pupil dilation, the derivative of the pupil size, and running speed. The directory contain NumPy arrays (of size `1 x 3`) where each single `X.npy` contains the behavioral variables (in the same order that was mentioned earlier) for trial `X`.
+  - **pupil_center:** the eye position of the mouse, estimated as the center of the pupil. The directory contain NumPy arrays (of size `1 x 2`) for horizontal and vertical eye positions.
 - **meta:** includes meta data of the experiment
     - **neurons:** This directory contains neuron-specific information. Below are a list of important variables in this directory
         - `area.npy`: contains the area of each neuron
-        - `cell_motor_coordinates.npy`: contains the position (x, y, z) of each neuron in the cortex
+        - `cell_motor_coordinates.npy`: contains the position (x, y, z) of each neuron in the cortex, given in microns. **Note:** The
```
Comment (mohammadbashiri) [inline notebooks/dataset_tutorial/README.md:17]: The text after "**Note:** The" is missing.

File: notebooks/dataset_tutorial/README.md [hunk 8]
```diff
@@ -1,21 +1,62 @@
...
-    - **trials:** This directory contains trial-specific meta data. Below are a list of important variables in this directory:
-        - `frame_image_id.npy`: contains unique image id. If the image is presented multiple times (which is the case in the test set) the image id does not change.
-        - `tiers.npy`: contains labels that are used to split the data into *train*, *validation*, and *test* set.
-        - `trial_idx.npy`: contains unique (hashed) index for each trial. While the true trial index is available for the “pre-training” datasets, it is hidden (i.e. hashed) in the competition datasets. As opposed to the image ids, the trial index is different for different trials even if the image stays the same.
\ No newline at end of file
+      - **Note:** The statistics of the responses are or particular importance, because we provide the deconvolved calcium traces here in the responses.
+      
+        However, for the evaluation of submissions in the competition, we require the responses to be **standardized** (i.e. `r = r/(std_r)`).
```
Comment (mohammadbashiri) [inline notebooks/dataset_tutorial/README.md:23]: Could you make it clear that the STD is computed across all images in the training set?

File: notebooks/submission_tutorial/README.md [hunk 9]
```diff
@@ -1,6 +1,5 @@
...
+# How to make a Submission
 
-Below we provide a brief explanation of our Baseline models
-
-- add links to publications (Lurz, Bashiri, Franke)
-- add link to Lurz Video
\ No newline at end of file
+- explain API
+- explain the logic behind storing the responses
+- stress the fact that responses in ground truth are standardized
```
Comment (mohammadbashiri) [inline notebooks/submission_tutorial/README.md:5]: Are these todos? or what is explained in the notebooks of this directory?

**Review by mohammadbashiri — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KonstantinWilleke) [reply README.md:32]: fixed

File: notebooks/model_tutorial/1a_model_training_sensorium.ipynb (hunk 2 above)
Comment (KonstantinWilleke) [reply notebooks/model_tutorial/1a_model_training_sensorium.ipynb:8]: fixed

**Comment by KonstantinWilleke** [comment]
<p>fixed</p>

---

 <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: notebooks/model_tutorial/0_baseline_CNN.ipynb (hunk 3 above)
Comment (KonstantinWilleke) [reply notebooks/model_tutorial/0_baseline_CNN.ipynb:2]: <p>fixed</p>

File: notebooks/model_tutorial/1b_model_training_sensorium+.ipynb (hunk 4 above)
Comment (KonstantinWilleke) [reply notebooks/model_tutorial/1b_model_training_sensorium+.ipynb:8]: <p>fixed</p>

**Comment by KonstantinWilleke** [comment]
<p>fixed</p>

---

 <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: notebooks/submission_tutorial/1_sensorium_submission.ipynb (hunk 5 above)
Comment (KonstantinWilleke) [reply notebooks/submission_tutorial/1_sensorium_submission.ipynb:1]: <p>fixed</p>

Comment (KonstantinWilleke) [reply notebooks/submission_tutorial/1_sensorium_submission.ipynb:1]: <p>fixed</p>

Comment (KonstantinWilleke) [reply notebooks/submission_tutorial/1_sensorium_submission.ipynb:1]: <p>fixed</p>

File: notebooks/dataset_tutorial/README.md (hunk 7 above)
Comment (KonstantinWilleke) [reply notebooks/dataset_tutorial/README.md]: fixed

File: notebooks/dataset_tutorial/README.md (hunk 8 above)
Comment (KonstantinWilleke) [reply notebooks/dataset_tutorial/README.md]: fixed

File: notebooks/submission_tutorial/README.md (hunk 9 above)
Comment (KonstantinWilleke) [reply notebooks/submission_tutorial/README.md]: I've extended it to its proper form.

**Review by mohammadbashiri — APPROVED** [review]
